uber/templates/base.html

```diff
@@ -471,7 +471,7 @@
       <script type="text/javascript">
           $(window).on("runJavaScript", function() {
               $('.form-control-static').each(function(index) {
-                  if($( this ).text().trim() == '') {
+                  if($( this ).text().trim() == '' && $(this).is(":visible")) {
```

Comment:
The attraction modal uses a very fancy confirmation box that loads an attendee's name in but is blank and hidden to start with. It being blank was tripping this function, which is intended to hide unfilled read-only text fields.

We probably need a better long-term solution to the fact that this function is so aggressive, but this is Good Enough For Now™